ctpolicy/chromepolicy.go

```diff
@@ -61,6 +62,44 @@ func (chromeP ChromeCTPolicy) LogsByGroup(cert *x509.Certificate, approved *logl
 	return groups, outerror
 }
 
+// LogsByGroup describes submission requirements for embedded SCTs according to
+// https://github.com/chromium/ct-policy/blob/master/ct_policy.md#qualifying-certificate.
+func (chromeP ChromeCTPolicy) LogsByGroup2(cert *x509.Certificate, approved *loglist2.LogList) (LogPolicyData, error) {
```

Comment:
Can this method return both err + data too?

Reply:
Added the explanation